examples/advanced/urdf_loading_and_manipulation/main.py

```diff
@@ -0,0 +1,66 @@
+import blenderproc as bproc
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('urdf_file', nargs='?', default="./model.urdf", help="Path to the .urdf file")
+parser.add_argument('output_dir', nargs='?', default="examples/advanced/urdf_loading_and_manipulation/output", help="Path to where the final files will be saved")
+args = parser.parse_args()
+
+bproc.init()
+
+robot = bproc.loader.load_urdf(urdf_file=args.urdf_file)
+
+robot.hide_irrelevant_objs()
```

Comment:
Should we maybe call that per default in `load_urdf`? Or is there a situation where you would want to render inertials and collision objects?

Reply:
Yes I think we could hide inertial and collision objects by default. They will still appear in the viewport if people want to debug.